spec/features/thredded/user_deletes_post_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+feature 'User deleting posts' do
+  scenario 'can delete their own post' do
+    user.log_in
+    topic = users_topic
+    topic.visit_topic
+    last_post = last_post(topic)
```

Comment:
I feel like `last_post` would be handy as part of that topic object ... eg: `last_post = topic.last_post`. If you move that method into the topic object and memoize it I could see it being used in the future.